apps/agreement/contracts/executor/IAgreementExecutor.sol

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-License-Identitifer:    GPL-3.0-or-later
+ */
+
+pragma solidity 0.4.24;
+
+import "@aragon/os/contracts/lib/token/ERC20.sol";
+
+
+interface IAgreementExecutor {
```

Comment:
If we're assuming this to be a forwarder, let's make it explicit with inheritance!

Reply:
We can probably turn this into a contract, cause interfaces cannot inherit in Solidity 0.4.24